### DISCUSSION TITLE
SH110X OLED Driver

### DISCUSSION THREAD

**Discussion by Reghardt** [microsoft/devicescript#630] [General]
Hello

I wrote a driver for the SH1106 and SH1107 OLED displays, which are quite common and similar to the SSD1306.
I would like to contribute this driver to Devicescript.

As I understand drivers fall under the category of built-in packages as per the contributions page?
May I create a pull request or should a discussion be setup first?

Kind regards
Reghardt

**Comment by pelikhan** [comment]
That's wonderful! I've added a page to help you with the process of adding a driver at https://microsoft.github.io/devicescript/developer/drivers/add-driver . This discussion is a perfect start.

**Reply by Reghardt** [reply]
Thank you very much I appreciate it!
I will get to it either tomorrow or the day after.
